hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/compress/lz4/TestLz4CompressorDecompressor.java

```diff
@@ -330,4 +328,33 @@ public void doWork() throws Exception {
 
     ctx.waitFor(60000);
   }
+
+  @Test
+  public void testLz4Compatibility() throws Exception {
+    Path filePath = new Path(TestLz4CompressorDecompressor.class
```

Comment:
nit: perhaps some comments on this - not quite sure what it is testing.

Reply:
ok, added some comment.